cockatrice/src/messagelogwidget.cpp

```diff
@@ -364,13 +364,20 @@ void MessageLogWidget::logMoveCard(Player *player,
 
 void MessageLogWidget::logDrawCards(Player *player, int number)
 {
+    int deckSize = player->getZones().value("deck")->getCards().size();
     soundEngine->playSound("draw_card");
     if (currentContext == MessageContext_Mulligan) {
         logMulligan(player, number);
     } else {
-        appendHtmlServerMessage(tr("%1 draws %2 card(s).", "", number)
+        if (deckSize == 0  && number == 0) {
+            appendHtmlServerMessage(tr("%1 had no cards left to draw.", "")
```

Comment:
```suggestion
            appendHtmlServerMessage(tr("%1 has no cards left to draw.")
```
most of our log messages use current tense instead of past tense, I think it'd make more sense to reword this in a way that works better in current tense, the original ticket has a suggestion in the way of "player draws from an empty library" or perhaps "player tries to draw from empty library", @tooomm might have some suggestions